tests/components/mikrotik/test_hub.py

```diff
@@ -65,9 +65,9 @@ async def test_hub_not_support_wireless(hass):
     hub = await setup_mikrotik_entry(hass, support_wireless=False)
 
     assert hub.api.devices["00:00:00:00:00:01"]._params == DHCP_DATA[0]
-    assert hub.api.devices["00:00:00:00:00:01"]._wireless_params is None
+    assert not hub.api.devices["00:00:00:00:00:01"]._wireless_params
```

Comment:
I'd remove this test module and replace it with tests in the `test_device_tracker.py` module instead, that don't access the coordinator directly.

Reply:
Done. please check and let me know if these tests are better.

Reply:
Looks good.